### PR TITLE
Rewordings for session 2

### PR DESCRIPTION
**Pull request by encukou**
I'd generally like to speak directly to the reader, in 2<sup>nd</sup> person, to make the text more personal.

### CODE REVIEW

File: lessons/beginners/print/index.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ na výrazy a jejich hodnoty podíváme podrobněji.
 > print(1, 2, 3)
 > ```
 >
-> Kolem operátorů jako `+` a `*` se obyčejně píše jedna mezera zleva a
+> Kolem operátorů jako `+` a `/` se obyčejně píše jedna mezera zleva a
```
Comment (hroncok) [inline lessons/beginners/print/index.md:149]: Just curious, what is the reason for this particular change?

Comment (encukou) [reply lessons/beginners/print/index.md:149]: To match the example just below

**Comment by encukou** [comment]
> I haven't checked if this is complete (e.g. if the entire chapter is in 2nd person)

I'll do another pass next semester :)

File: lessons/beginners/print/index.md (hunk 1 above)
Comment (encukou) [reply lessons/beginners/print/index.md:149]: ... and as to why use that example, ½ looks tighter than 2·2 or other expressions I could find, so it looks best without spaces.

**Comment by encukou** [comment]
Thank you for the review!

**Pull request merged**